lib/charms/opensearch/v0/opensearch_relation_peer_cluster.py

```diff
@@ -164,6 +165,7 @@ def _on_peer_cluster_relation_changed(self, event: RelationChangedEvent):
 
         if not (candidate_failover_app := data.get("candidate_failover_orchestrator_app")):
             self.refresh_relation_data(event)
+            event.defer()
```

Comment:
I don't think that's needed, we don't have to have a failover, a main orchestrator is plenty and the only required orchestrator.